api/namex/resources/name_requests/name_request.py

```diff
@@ -267,20 +267,37 @@ def patch(self, nr_id, nr_action):
             nr_svc.nr_num = nr_model.nrNum
             nr_svc.nr_id = nr_model.id
 
-            valid_states = State.VALID_STATES
+            # Only allow editing if the request is in certain valid states
+            request_editable_states = [
+                State.DRAFT,
+                State.RESERVED,
```

Comment:
Why is reserved and cond-reserve editable.  We dont want to edit those from the patch?

Reply:
Maybe editable isn't the best word... I'm just checking to see what states should actually be 'handled' within the PATCH method.